docs/core/diagnostics/runtime-interop-events.md

```diff
@@ -0,0 +1,32 @@
+
+# Interop Events
+
+These runtime events capture information about Microsoft intermediate language (MSIL) stub generation.  
+
+This category consists of the following event:
+
+- [ILStubGenerated](#ilstubgenerated-event)
+
+## ILStubGenerated Event
+
+|Keyword for raising the event|Level|  
+|-----------------------------------|-----------|  
+|`InteropKeyword` (0x2000)|Informational(4)|  
+  
+|Event|Event ID|Raised when|  
+|-----------|--------------|-----------------|  
+|`ILStubGenerated`|91|An IL Stub is generated.|
```

Comment:
This event is `88` I believe. I would also document the `ILStubCacheHit` event.

https://github.com/dotnet/runtime/blob/73e1543b191093154b940853b30296a6a1a3f145/src/coreclr/src/vm/ClrEtwAll.man#L3385-L3394

Reply:
> I would also document the ILStubCacheHit event.

Thanks for the feedback @AaronRobinsonMSFT! So I did notice the `ILStubCacheHit` payload description in `ClrEtwAll.man` but I didn't see the being fired from the runtime when I searched for it, which is why I didn't document it. If this event still is being fired, could you please point me to the code where it's being fired from?

Reply:
Oops. Looks like that was indeed removed when moving from .NET Core. I am going to add this back in .NET 6 because it does have value for us. I will add you on the PR when it goes up.

Reply:
Sounds great! Thanks :)

Reply:
@sywhang FYI: https://github.com/dotnet/runtime/pull/42881

Reply:
@sywhang Sorry about this. Seems we have decided to not add back the `ILStubCacheHit` event. Its addition can be reverted. Thanks for indulging me.